src/branch/branch.repository.ts

```diff
@@ -115,4 +115,15 @@ export class BranchRepository {
 
 		return result ? mapTableRecordToReadModel(result) : undefined;
 	}
+
+	async reject(branchId: number) {
+		await this.dbClient
+			.db()
+			.updateTable('branch')
+			.set({
+				status_id: 2,
```

Comment:
You can replace hardcoded status ID with ` BRANCH_STATUS_TO_ID` function from branch.ts file:

```
export const BRANCH_STATUS_TO_ID = (status: BranchStatus) => {
	switch (status) {
		case 'active':
			return 1;
		case 'rejected':
			return 2;
		case 'pending':
			return 3;
		default:
			throw new Error('Invalid branch status');
	}
};
```

Example usage:
`status_id: BRANCH_STATUS_TO_ID(BranchStatus.Rejected)`